PowerEditor/src/WinControls/OpenSaveFileDialog/CustomFileDialog.cpp

```diff
@@ -50,7 +50,7 @@ namespace // anonymous
 
 	static const int IDC_FILE_CUSTOM_CHECKBOX = 4;
 	static const int IDC_FILE_TYPE_CHECKBOX = IDC_FILE_CUSTOM_CHECKBOX + 1;
-	static const TCHAR TEMP_OK_BUTTON_LABEL[] = _T("Save");
+	static const TCHAR TEMP_OK_BUTTON_LABEL[] = _T("Open");
```

Comment:
3 commits seems like a lot to change "save" to "open" :-)

Reply:
Sorry, but no. This won't work.
1. Take a look at Save dialog.
2. Change OS language and take a look at the button.

Reply:
The Save dialog has no issues when I checked, shows "Save" label.